jest.config.js

```diff
@@ -42,6 +42,7 @@ const getConfig = baseConfig => ({
       lines: 75,
     },
   },
+  testPathIgnorePatterns: ['<rootDir>/dist/'],
```

Comment:
If the dist directory is built, this prevents jest running the tests in there too!